src/installer/tests/Assets/TestProjects/TestWindowsOsShimsApp/TestWindowsOsShimsApp.csproj

```diff
@@ -7,6 +7,7 @@
     <RuntimeFrameworkVersion>$(MNAVersion)</RuntimeFrameworkVersion>
     <DefineConstants Condition="'$(OS)' == 'Windows_NT'">WINDOWS;$(DefineConstants)</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <NETCoreAppMaximumVersion>6.0</NETCoreAppMaximumVersion>
```

Comment:
You should not hardcode these values here. Instead you want to add this line to `TestProjects.targets` which is imported here and in other projects:

```
  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory),global.json))\Directory.Build.targets" />
```